src/components/UserProfile/TeamWeeklySummaries/TeamWeeklySummaries.css

```diff
@@ -27,3 +31,24 @@
   align-items: center;
   line-height: 0.8;
 }
+
+/* .copy-icon {
+  /* cursor: pointer;
+  position: absolute;
+  right: 0;
+  top: 0;
+  margin: 2px 2px; */
+
+.copy-icon {
+  margin: 2px 2px;
+}
+.team-weekly-summary-container {
+  display: flex;
+  flex-direction: row;
+  align-items: start;
```

Comment:
Change for:

align-items: flex-start;

Reply:
Hello @yurisokolovicz
I've fixed the code with your suggestion. 
Thanks!

Reply:
Thank you very much!